### ISSUE TITLE
Does not detect emulators on Windows due to '.exe' extension

### ISSUE THREAD

**Issue by DLehenbauer** [empathicqubit/vscode-cc65-debugger#90]
2/3 of my Windows 10 machines could not launch xPet.  The 3rd machine worked no problem.

On the effected machines, pressing F5 displayed a dialog saying "Error: Couldn't find emulator..." etc. that showed the correct path to xPet, but without the *.exe extension.

However, after modifying the extension by adding the ".exe" to the emulator's filename in 214.js the extension began working:
```js
MachineType.pet?"xpet.exe"     // <-- Add '.exe' (around character 66606)
```
Here is how the machines were set up:

Machine 1 (did not work)
- Clean Windows 10 install
- Installed VSCode
- Installed vscode-cc64-debugger from marketplace
- Installed choco
- choco install winvice-nightly
- choco install make
- choco install git

Machine 2 (did not work) and Machine 3 (worked) were both old Windows 10 / VSCode installations with Vice 3.5 installed from the official sourceforge binary release (unpacked *.7z).

I can't think of a reason machine 3 would work while the other two machines are picky about *.exe extension.  In all cases, I was using the assembly language template from this repo.

(Great project, BTW!)

**Comment by empathicqubit** [comment]
Thanks for pointing this out! Would it be possible for you to submit a patch for this? Seem like you're most of the way there. I'm going to be busy over the next few weeks so I probably won't have much time to look at it.

**Comment by DLehenbauer** [comment]
Sure, I'll take a look.  :-)

**Comment by empathicqubit** [comment]
Your fix is in v5.1.3. Thanks for your code contribution!
